.gitignore

```diff
@@ -85,3 +85,5 @@ examples/*/esp32/dependencies.lock
 # jupyter temporary files
 .ipynb_checkpoints
 
+# py environment folder
+py
```

Comment:
```suggestion
py
__pycache__
```

Reply:
That's interesting, __pycache__ was already in the file, add it again?